### PR TITLE
Fix classhash placeholder path

### PR DESCRIPTION
**Pull request by rodrigo-pino**

### CODE REVIEW

File: src/passes/newToDeploy.ts [hunk 1]
```diff
@@ -50,8 +50,9 @@ import { getContainingSourceUnit } from '../utils/utils';
  *
  *   Since solidity stores the bytecode of the contract to create, and cairo uses
  *   a declaration address(`class_hash`) of the contract on starknet and the latter
- *   can only be known when interacting with starknet, empty placeholders are used
- *   for compiling, and then filled during deployement with the right address.
+ *   can only be known when interacting with starknet, placeholders are used to
+ *   store this hashes, nonetheless at this stage these hashes cannot be known
```
Comment (cicr99) [inline src/passes/newToDeploy.ts:54]: `this` -> `these`

**Comment by swapnilraj** [comment]
I see the salt bytes size is changed,
this will need adjustment as well then
https://github.com/NethermindEth/UniStark/blob/main/test/shared/utilities.ts#L50

File: tests/behaviour/expectations/types.ts [hunk 2]
```diff
@@ -16,10 +18,10 @@ export class AsyncTest {
     return `${this.name}.sol`;
   }
   get cairo() {
-    return `warp_output/${mangleContractFilePath(manglePath(this.name), this.contract)}.cairo`;
+    return `${OUTPUT_DIR}/${mangleContractFilePath(manglePath(this.name), this.contract)}.cairo`;
```
Comment (swapnilraj) [inline tests/behaviour/expectations/types.ts:21]: minor nit: path.resolve or path.join instead of /

File: src/passes/warnSupportedFeatures.ts [hunk 3]
```diff
@@ -50,6 +75,15 @@ export class WarnSupportedFeatures extends ASTMapper {
       [...addresses.entries()].forEach(([path, nodes]) => warn(path, nodes));
     }
 
+    if (deploySalts.size > 0) {
+      console.log(
+        `${warning(
+          'Warning',
```
Comment (swapnilraj) [inline src/passes/warnSupportedFeatures.ts:81]: @rodrigo-pino can we make a note of this in the docs as well

**Review by swapnilraj — APPROVED** [review]

File: src/passes/warnSupportedFeatures.ts (hunk 3 above)
Comment (rodrigo-pino) [reply src/passes/warnSupportedFeatures.ts:81]: There was already mention of it in `doc/features/contract_factories.mdx` so I updated that part

**Review by JorikSchellekens — APPROVED** [review]

**Pull request merged**